randovania/games/dread/json_data/Ferenia.txt

```diff
@@ -2942,23 +2949,17 @@ Extra - asset_id: collision_camera_034
       All of the following:
           After Ferenia - db_reg_b2_007 and After Ferenia - db_reg_b2_008 and After Ferenia - db_reg_b2_009
           Gravity Suit or After Ferenia - Wave Emmi Lower Water
-          Space Jump or Simple IBJ
-  > Tunnel to Central Unit Access
-      All of the following:
-          Before Ferenia - Central Unit
-          Gravity Suit or After Ferenia - Wave Emmi Lower Water
-          Space Jump or Infinite Bomb Jump (Beginner)
+  > Tunnel to Central Unit Access (Top)
+      Any of the following:
+          All of the following:
+              Gravity Suit and Before Ferenia - Wave Emmi Lower Water
+              Space Jump or Walljump (Intermediate) or Simple IBJ
+          Speed Booster and Speedbooster Conservation (Intermediate) and Disabled Door Lock Randomizer
   > Lower-Right Alcove
       After Ferenia - Central Unit and Can Slide
 
 > Lower-Left Alcove; Heals? False
   * Layers: default
-  > Start Point
-      All of the following:
-          Morph Ball and After Ferenia - Central Unit
-          Any of the following:
-              Simple IBJ or Use Spin Boost
-              Slide and Slide Jump (Intermediate) and Walljump (Intermediate)
```

Comment:
why did you remove this connection?

Reply:
Added it back in. Don´t know y but I saw the Flashgates in reverse